projects/extension/src/hooks/useActiveChains.ts

```diff
@@ -120,30 +115,20 @@ const createChainDetailObservable = (chain: PageChain) =>
     type ChainHead$ = ReturnType<(typeof observableClient)["chainHead$"]>
     let chainHead: ChainHead$
     let unfollow: ChainHead$["unfollow"]
-    const followWithRetry$ = defer(() => {
+    const followAndBestBlocksWithRetry$ = defer(() => {
       chainHead = observableClient.chainHead$()
       unfollow = chainHead.unfollow
-      return chainHead.follow$
+      return chainHead.follow$.pipe(
+        combineLatestWith(chainHead.bestBlocks$.pipe(startWith([]))),
+      )
     }).pipe(
       catchError(() => EMPTY),
       repeat({ delay: 1 }),
       share(),
     )
```

Comment:
@josepot Would you check this?

The goal is to re-subscribe to `chainHead.follow$` and `chainHead.bestBlocks$` when there is an error